### PR TITLE
PANOS connections per second

### PR DESCRIPTION
**Pull request by systeembeheer-rtvu**
Added Palo Alto Networks PANOS 9.0.x MIB files
Added graphs for connections per second for vsys 1
Added graphs for connections per second to device firewall overview

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by systeembeheer-rtvu** [comment]
The Travis CI check failed. Is there something I can do to fix this?

**Comment by PipoCanaja** [comment]
Hi @systeembeheer-rtvu 
- Could you remove the file "includes/definitions/discovery/test-os.yaml" ? This is a leftover from tests I suppose :) 
- For the Travis error, we need help, I am not familiar at all with it for the moment. @murrant , if you come around this PR, we need you ;)

**Comment by systeembeheer-rtvu** [comment]
I'm not sure what information I fail to deliver. @murrant Can you put me on the right track?

**Comment by Jellyfrog** [comment]
@systeembeheer-rtvu could you please rebase your branch? you have a merge conflict

File: misc/config_definitions.json [hunk 1]
```diff
@@ -2003,6 +2003,37 @@
                 "descr": "Active UDP Sessions"
             }
         },
+
+ 	"graph_types.device.panos_vsys_active_udp_cps": {
+            "default": {
+                "section": "firewall",
+                "order": 0,
+                "descr": "UDP connections p/s",
+		"type": "integer"
```
Comment (Jellyfrog) [inline misc/config_definitions.json]: `type` should be outside of `default`

File: misc/config_definitions.json [hunk 2]
```diff
@@ -2003,6 +2003,37 @@
...
+
+            }
+	},
+
+ 	"graph_types.device.panos_vsys_active_tcp_cps": {
+            "default": {
+                "section": "firewall",
+                "order": 0,
+                "descr": "TCP connections p/s",
+		"type": "integer"
```
Comment (Jellyfrog) [inline misc/config_definitions.json]: same as above

File: misc/config_definitions.json [hunk 3]
```diff
@@ -2003,6 +2003,37 @@
...
+
+            }
+	},
+
+ 	"graph_types.device.panos_vsys_active_other_ip_cps": {
+            "default": {
+                "section": "firewall",
+                "order": 0,
+                "descr": "Other connections p/s",
+		"type": "integer"
```
Comment (Jellyfrog) [inline misc/config_definitions.json]: same as above

File: includes/definitions/discovery/panos.yaml [hunk 4]
```diff
@@ -37,4 +37,4 @@ modules:
                     num_oid: '.1.3.6.1.4.1.25461.2.1.2.3.{{ $index }}'
                     group: Sessions
                     descr: Active Sessions
-                    high_limit: panSessionMax
+                    high_limit: panSessionMax
```
Comment (Jellyfrog) [inline includes/definitions/discovery/panos.yaml]: Please config your editor to insert newline at end of file

**Comment by murrant** [comment]
Should these be count sensors?

**Comment by systeembeheer-rtvu** [comment]
> Should these be count sensors?

Murrant, the lingo is rather new to me. I suppose these are counters as they count sessions per second. How should I interpret 'sensors'?

**Comment by murrant** [comment]
https://docs.librenms.org/Developing/os/Health-Information/

When you add as described in this doc, it allows for alerting and other nice things.

**Comment by murrant** [comment]
Also, having them be count sensors includes the ability to alert on the values.

**Comment by systeembeheer-rtvu** [comment]
I'll have to read up on that. If anyone more fluent with Librenms want to chime in, drop me a line. Happy to cooperate.

**Comment by systeembeheer-rtvu** [comment]
The "sessions per second" should probably be a gauge and not a counter. Can we alert on gauges?

**Comment by PipoCanaja** [comment]
"count" type sensor is providing the exact value received via SNMP. So basically, it is a GAUGE for RRD, not a COUNTER or DERIVE ... So you are safe to go with a "count" sensor.

**Comment by systeembeheer-rtvu** [comment]
In __includes/definitions/discovery/panos.yml__ I've found the folowing:
```
count:
            data:
                -
                    oid: panSession
                    value: panSessionActive
                    num_oid: '.1.3.6.1.4.1.25461.2.1.2.3.{{ $index }}'
                    group: Sessions
                    descr: Active Sessions
                    high_limit: panSessionMax
                    index: 'panSessionActive.{{ $index }}'
```
Based on the example in https://docs.librenms.org/Developing/os/Health-Information/ I've added 
```
"index: 'panSessionActive.{{ $index }}' 
```

Either I'm missing the point or the requested counters were already in place. Some guidance on how to test this is appreciated.

**Comment by murrant** [comment]
Fixed by #11341

**Pull request closed without merging**